blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -89,401 +33,93 @@ export class RunDetailsPipeline extends Component {
             document.addEventListener('keydown', this._handleKeys, false);
```

Comment:
I am not 100% sure but the behaviour is still the same as in current implementation in aspect of stopping.

I tried 

```
window.addEventListener('scroll', (event) => console.log(event), false);
```

but this did not kicked in.

I followed @cliffmeyers and attached it to `document` and dropped the `ref`